asgard/direct_path_response_builder.cpp

```diff
@@ -529,6 +535,14 @@ void set_path_item_instruction(const DirectionsLeg_Maneuver& maneuver, pbnavitia
     }
 }
 
+void set_path_item_instruction_start_coords(pbnavitia::PathItem& path_item, const valhalla::midgard::PointLL& instruction_start_coords) {
+    if (!path_item.instruction().empty()) {
+        auto* coords = path_item.mutable_instruction_start_coordinates();
+        coords->set_lat(instruction_start_coords.lat());
+        coords->set_lon(instruction_start_coords.lng());
+    }
```

Comment:
https://stackoverflow.com/questions/268132/invert-if-statement-to-reduce-nesting

```suggestion
    if (!path_item.instruction().empty()) { return; }
    
    auto* coords = path_item.mutable_instruction_start_coordinates();
    coords->set_lat(instruction_start_coords.lat());
    coords->set_lon(instruction_start_coords.lng());
```